content/posts/analytics/index.fr.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Analytiques"
+date: 2020-06-08T06:00:23+06:00
+description: Ajouter l'analytique dans hugo theme Toha
+menu:
```

Comment:
Please, add yourself as post author.

Reply:
For me, it's not mandatory about this. But it's done.